bouncer/shared/utils.ts

```diff
@@ -276,6 +276,7 @@ function getCachedSubstrateApi(defaultEndpoint: string) {
                 setTimeout(() => {
                   if (connections === 0) {
                     api = undefined;
+                    /* eslint-disable-next-line @typescript-eslint/no-floating-promises */
                     Reflect.get(target, 'disconnect', receiver).call(target);
```

Comment:
I'd suggest adding a dummy catch, because this is in a `setTimeout` there's no way to handle it throwing. The idea here is that between tests we want to wait 5 seconds to actually disconnect to prevent all the subscribers from unsubscribing and then immediately opening a new subscription
```suggestion
                    Reflect.get(target, 'disconnect', receiver).call(target).catch(() => null);
```